ios/purchases_flutter.podspec

```diff
@@ -16,6 +16,10 @@ Pod::Spec.new do |s|
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
   s.dependency 'PurchasesHybridCommon', '1.4.2'
+  s.pod_target_xcconfig   = { 'DEFINES_MODULE' => 'YES' }
```

Comment:
what does this do?

Reply:
I created a new Flutter plugin and they do this by default:
```
 # Flutter.framework does not contain a i386 slice. Only x86_64 simulators are supported.
  s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES', 'VALID_ARCHS[sdk=iphonesimulator*]' => 'x86_64' }
```

Reply:
it enables stricter header search paths, meaning that hopefully importing errors would surface before being able to publish the pod. 
it also defines a Module, which allows for the semantic import syntax, i.e.: `@import Purchases`. 
I'll add the VALID_ARCHS thing 👍